test/cql-pytest/test_using_timestamp.py

```diff
@@ -90,3 +90,56 @@ def test_key_writetime(cql, table1):
         cql.execute(f'SELECT writetime(k) FROM {table1}')
     with pytest.raises(InvalidRequest, match='PRIMARY KEY part k'):
         cql.execute(f'SELECT ttl(k) FROM {table1}')
+
+# Reproducer for https://github.com/scylladb/scylladb/issues/14182
+def test_rewrite_using_same_timeout_and_ttl(scylla_only, cql, table1):
```

Comment:
Wait, what?
I thought the main point of this patch was to behave like Cassandra always did in this specific case. So why a scylla_only test case?

Reply:
I wasn't sure at the time that we're indeed compatible.
Today I'm more confident that we are (with recent enough version of Cassandra).
I'll test that in the next version of this series.